client/src/Views/ReportsPage/ReportsPage.vue

```diff
@@ -1,40 +1,56 @@
 <template>
   <div class="reports-container">
     <template class="reports-container__v-tabs">
-      <v-tabs fixed-tabs v-model="tabs" :class="tabBarColoring">
-        <v-tab>Monthly Reports</v-tab>
-        <v-tab>Users</v-tab>
-        <v-tab>All Shouts</v-tab>
+      <v-tabs fixed-tabs v-model="activeTab" :slider-color="tabBarColoring">
+        <v-tab
+          v-for="tab in tabs"
+          :exact-active-class="tab.activeClass"
+          :key="tab.name"
+          :to="tab.route"
+          exact
+          >{{ tab.name }}</v-tab
+        >
       </v-tabs>
     </template>
     <div class="reports-container__main">
-      <v-tabs-items v-model="tabs">
-        <MonthlyReports />
-        <UserReports />
-        <AllShouts />
-      </v-tabs-items>
+      <router-view />
     </div>
   </div>
 </template>
 
 <script>
-import MonthlyReports from './MonthlyReports';
-import UserReports from './UserReports';
-import AllShouts from './AllShouts';
-
 export default {
-  name: 'Reports',
-  components: { MonthlyReports, UserReports, AllShouts },
+  name: "Reports",
   data() {
     return {
-      tabs: null,
+      activeTab: "/reports",
```

Comment:
Based on the way that Techtonic' site handles the dropdowns, I think that not having a default / clickable link but rather force them to choose one of the links in the dropdown would probably be a better approach?